README.md

```diff
@@ -125,4 +125,4 @@ Feel free to check the [issues page](https://github.com/Olivier-Kango/vet_clinic
 This project is [MIT](./LICENSE.txt) licensed.
 
 
```

Comment:
- [ ] `data.sql file`: Kindly make sure  you add all the code lines in the `data.sql` file you used while adding data you were required in the **[project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md)**

![Screenshot from 2022-12-15 20-28-20](https://user-images.githubusercontent.com/83042742/207939905-466129c3-5228-49e4-8957-a4f540fcce1f.png)